shared/chat/conversation/list.desktop.js

```diff
@@ -339,7 +346,17 @@ class ConversationList extends Component<void, Props, State> {
     const isFirstMessage = index === 0
     const isSelected = message.messageID != null && this.state.selectedMessageID === message.messageID
 
-    const options = this.props.optionsFn(message, prevMessage, isFirstMessage, isSelected, isScrolling, key, style, this._onAction)
+    const onShowEditor = (message: Message, event: any) => {
```

Comment:
Can this be defined as a method of the component so that the closure isn't recreated for every row?